ext/maxminddb.c

```diff
@@ -405,11 +467,29 @@ static zend_class_entry *lookup_class(const char *name TSRMLS_DC)
         zend_error(E_ERROR, "Class %s not found", name);
     }
     return *ce;
+#endif
 }
 
+#ifdef ZEND_ENGINE_3
```

Comment:
These two function are almost identical. It might be simpler to just share the code.